Makefile

```diff
@@ -36,7 +36,7 @@ PKG          := ./...
 TAGS         :=
 TESTS        := ".*"
 TESTTIMEOUT  := 15s
-RACETIMEOUT  := 1m
```

Comment:
It's at or near 1m without this change. I've seen it take 1m:22s and 1m:0s with this change.